connectors/csws/connector/src/main/java/org/apache/manifoldcf/csws/CswsSession.java

```diff
@@ -613,12 +628,17 @@ private void processIOException(IOException e)
   
   private void processSOAPFault(SOAPFaultException e)
     throws ManifoldCFException, ServiceInterruption {
-    throw new ManifoldCFException("SOAP exception: "+e.getMessage(), e);
-    // MHL
+    // Repeat on SOAP Errors
```

Comment:
hard to say, if the other extreme (fail job on all occasions vs. retry on all occasions) is better